pool/app/pool_common/locales/i18n_de.ml

```diff
@@ -342,11 +346,10 @@ Wenn Sie die Erinnerungen jetzt manuell auslösen werden über den gewählten Na
      Testnachricht schicken können, um den API Key zu verifizieren. Die Nummer \
      muss im Format +41791234567 sein."
   | I18nText str -> str
-  | MissingMessageTemplates (label, languages) ->
-    Format.asprintf
-      "Das '%s' Template fehlt in den folgenden Sprachen: %s"
-      label
-      (languages |> CCString.concat ", ")
+  | MissingMessageTemplates ->
+    "Die folgenden Nachrichtenvorlagen sind nicht vorhanden. Die \
```

Comment:
welche folgenden? (german and english)

Reply:
Diese: https://github.com/uzh/pool/pull/279/files#diff-5958c332d013c1af8cf8f84f61856eee77ff6129afedb5864a91bd23e62f3fa4R62

Reply:
a list called list^^